lib/util.js

```diff
@@ -181,9 +181,10 @@ inspect.colors = {
   'red': [31, 39],
   'yellow': [33, 39]
 };
+inspect.colors = Object.assign(Object.create(null), colors);
```

Comment:
@nem035 You can make this change if it makes sense to you, there are no fixed rules so as long as nobody explicitly *objects* to the current state of the PR.

Fwiw I’d be in favour of @TimothyGu’s suggestion.